### ISSUE TITLE
forms with style (Gyfter, Gyfted, Deliverer)

### ISSUE THREAD

**Issue by jasonleonhard** [jasonleonhard/gyfted#12]

**Comment by jasonleonhard** [comment]
As we discussed in person, take a peek at WTForm to use less code and write clean forms and validations. In particular, you might google {{wtf.quick_form(form)}} 
Best!

**Comment by jasonleonhard** [comment]
If we use WTForm we can leverage their validations. Feel free to look into this.

**Comment by 91ashika** [comment]
Yup I checked it. But the forms created using quickform() cannot be customized according to our preference. It is automatically generated.
